controllers/deployment.go

```diff
@@ -35,8 +41,21 @@ func (r *DeploymentReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *DeploymentReconciler) Reconcile(ctx context.Context, application *skiperatorv1alpha1.Application) (reconcile.Result, error) {
 	application.FillDefaults()
 
+	gcpIdentityConfigMap := corev1.ConfigMap{}
+
+	err := r.client.Get(ctx, types.NamespacedName{Namespace: "skiperator-system", Name: "gcpidentityconfig"}, &gcpIdentityConfigMap)
+	if errors.IsNotFound(err) {
+		r.recorder.Eventf(
+			application,
+			corev1.EventTypeWarning, "Missing",
+			"Cannot find configmap named gcpidentityconfig in namespace skiperator-system",
```

Comment:
Can we rename the configmap to `gcp-identity-config` to make it easier to read?
```suggestion
			"Cannot find configmap named gcp-identity-config in namespace skiperator-system",
```

Reply:
changed name of configmap